### PR TITLE
workflow: move workflow execution to server side

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

File: rbac/policy.csv [hunk 1]
```diff
@@ -21,6 +21,8 @@ p, admin, noderule, read, allow
 p, admin, noderule, write, allow
 p, admin, edgerule, read, allow
 p, admin, edgerule, write, allow
+p, admin, workflowcall, read, allow
```
Comment (lebauce) [inline rbac/policy.csv]: this one doesn't seem to be used

File: rbac/policy.csv [hunk 2]
```diff
@@ -21,6 +21,8 @@ p, admin, noderule, read, allow
...
+p, admin, workflowcall, write, allow
```
Comment (lebauce) [inline rbac/policy.csv]: workflow.call according to the suggest route

File: api/server/workflow_call.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	}
+	return workflow.(*types.Workflow), nil
+}
+
+func (wc *WorkflowCallAPIHandler) registerEndPoints(s *shttp.Server, authBackend shttp.AuthenticationBackend) {
+	routes := []shttp.Route{
+		{
+			Name:        "WorkflowCall",
+			Method:      "POST",
+			Path:        "/api/workflowcall",
```
Comment (lebauce) [inline api/server/workflow_call.go]: I would prefer that we use /api/workflow/<id>/call

Comment (masco-enovance) [reply api/server/workflow_call.go]: done.

File: rbac/policy.csv (hunk 1 above)
Comment (masco-enovance) [reply rbac/policy.csv]: removed

File: rbac/policy.csv (hunk 2 above)
Comment (masco-enovance) [reply rbac/policy.csv]: done

File: api/server/runtime.go [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+		data := []byte(call.Argument(2).String())
+
+		subs := strings.Split(url, "/") // filepath.Base(url)
+		if len(subs) < 3 {
+			return r.MakeCustomError("WrongArgument", fmt.Sprintf("Malformed URL %s", url))
+		}
+		resource := subs[2]
+
+		// For topology query, we directly call the Gremlin engine
+		if resource == "topology" {
```
Comment (safchain) [inline api/server/runtime.go]: this is not really related to this patch but I don't understand the purpose of this :)
@lebauce any idea ?

Comment (safchain) [reply api/server/runtime.go]: ok got it , tricky thing :)
